front/app/components/admin/PostManager/components/FilterSidebar/FilterRadioButton/LabelContentWrapper.tsx

```diff
@@ -13,11 +13,6 @@ const LabelContentWrapper = ({ children }: Props) => {
       width="100%"
       justifyContent="space-between"
       alignItems="center"
-      // This is needed to ensure buttons of the same height.
-      // The 'All filters' button, for example, is less tall than the phase filters
-      // that have a circled phase number.
-      // (This could perhaps be moved to phase filter specific components if it turns out the others don't need it.)
-      height="24px"
     >
```

Comment:
I checked the different tabs/filters in various input managers and didn't spot any issues :)